src/graphiql.ts

```diff
@@ -40,12 +40,12 @@ export function renderGraphiQL(_req: Request, res: Response): void {
       crossorigin="anonymous"
     ></script>
     <script
-      src="https://unpkg.com/graphiql/graphiql.min.js"
-      integrity="sha384-JHFGCFTH/jUGV6uL38eHuw/3kb0aUiF0EzWPP68NQxvz7ldZXEoCv9RBGtsFHM86"
+      src="https://unpkg.com/graphiql@2.4.0/graphiql.min.js"
+      integrity="sha384-MVcRONLrOPBea05C9lB5pu31zTbPkbZI/gIn2y0r5xH6GefRkJcnrBGlY4zo2nwC"
       crossorigin="anonymous"
     ></script>
     <script
-      src="https://unpkg.com/@graphiql/plugin-explorer@0.1.12/dist/graphiql-plugin-explorer.umd.js"
+      src="https://unpkg.com/@graphiql/plugin-explorer@0.1.14/dist/graphiql-plugin-explorer.umd.js"
       integrity="sha384-rbN7vYh67VVbUGRkgCVTFu0Pk25KR5Ns9b6sOyPHf5GiYat1ce44UEF5I+HqE2Dg"
```

Comment:
Looks like there were no changes after all...

Reply:
Could be dev dependencies which doesn't change release build